docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,95 @@ weight = 4
 
 # Playlist
 
-The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.
+The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, and show off your metrics to your team or visitors.
 
-Since Grafana automatically scales Dashboards to any resolution they're perfect for big screens!
+Grafana automatically scales Dashboards to any resolution, which makes them perfect for big screens.
+
+Access the Playlist feature from Grafana's side menu, in the Dashboard submenu.
 
 ## Create a playlist
 
 {{< docs-imagebox img="/img/docs/v50/playlist.png" max-width="25rem" class="docs-image--right">}}
 
-The Playlist feature can be accessed from Grafana's sidemenu, in the Dashboard submenu.
+To create a playlist: 
+
+1. Click the __New playlist__ button.
```

Comment:
I think we should use double asterisks for bold text to align with the GitHub editors. I've proposed it in a future style guide here: https://github.com/grafana/grafana/issues/19592

Reply:
Also, we should use _lazy numbering_ for lists:

```
1. First
1. Second
1. Third
```

which will be displayed as:

```
1. First
2. Second
3. Third
```

This is great for when you want to insert an instruction in the middle of the list for example.

Reply:
If there are no existing playlists since before, the button will say **Create Playlist**:

<img width="953" alt="Screen Shot 2019-10-03 at 11 17 46" src="https://user-images.githubusercontent.com/8396880/66116695-75e03980-e5d3-11e9-8fe0-ca6c39f987bb.png">

Reply:
Got it on the lazy numbering! 

But for the asterisks, I used underscores because things do not bold in your Grafana docs with asterisks, as seen in my previous edits. Instead, what does display in bold has underscores. Example: https://github.com/grafana/grafana/issues/19341

If you still would like them with asterisks, I will change them right away.

Reply:
Interesting! I'll see if I can verify that double asterisks don't work. I'll get back to you on this

Reply:
In my update (after going through your edits, I've left this as-is. Will change if you require it, after you check on it.

Reply:
I can confirm double asterisks work when the docs are generated. So you can go ahead and update them 👍

Reply:
Sounds good! These have all been updated.